HACKING.md

```diff
@@ -105,7 +105,7 @@ The API is just JSON files generated in the `api` directory by `_plugins/create-
 
 ### API Documentation
 
-The API Documentation is available at <https://endoflife.date/docs/api> and is generated from an OpenAPI Specification file located at `_data/openapi.yml`. The documentation is rendered [Stoplight Elements](https://meta.stoplight.io/docs/elements/ZG9jOjMyNjU4OTY0-introduction-to-elements).
+The API Documentation is available at <https://endoflife.date/docs/api/v1/> and is generated from an OpenAPI Specification file located at `_data/openapi.yml`. The documentation is rendered [Stoplight Elements](https://meta.stoplight.io/docs/elements/ZG9jOjMyNjU4OTY0-introduction-to-elements).
```

Comment:
```suggestion
The API Documentation is available at <https://endoflife.date/docs/api/v1/> and is generated from an OpenAPI Specification file located at `_data/openapi.yml`. The documentation is rendered by [Stoplight Elements](https://meta.stoplight.io/docs/elements/ZG9jOjMyNjU4OTY0-introduction-to-elements).
```

Reply:
In fact the entire paragraph must be updated. Done.